MapboxCoreNavigation/RouteController.swift

```diff
@@ -267,7 +272,24 @@ open class RouteController: NSObject {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
     }
+    
+    func didChangeOrientation() {
+        if let previousOrientationChangeToPortrait = sessionState.previousOrientationChangeToPortrait, UIDevice.current.orientation == .portrait {
+            sessionState.timeInPortrait += previousOrientationChangeToPortrait.timeIntervalSinceNow
+        }
+        
+        sessionState.previousOrientationChangeToPortrait = Date()
```

Comment:
What happens if I'm in portrait, go into the background, rotate to landscape, and reopen the app?